modules/new_profile/php/profile.class.inc

```diff
@@ -0,0 +1,176 @@
+<?php
```

Comment:
Hi @kongtiaowang a suggestion in another PR from @johnsaigle is that we use `<?php declare(strict_types=1);` here instead of `<?php`.